src/app/dashboard/components/chart/index.tsx

```diff
@@ -21,11 +21,14 @@ type Props = {
 
 const StockChart: React.FC<Props> = ({ data }) => {
   const theme = useTheme();
-  const isAggregate = data[0].low !== undefined && data[0].high !== undefined;
+  const isAggregate =
+    data.length !== 0 &&
+    data[0].low !== undefined &&
+    data[0].high !== undefined;
   const lineType = 'linear';
 
-  const lowestChartValue = Math.min(...data.map((d) => d.low ?? d.avg));
-  const highestChartValue = Math.max(...data.map((d) => d.high ?? d.avg));
+  const lowestChartValue = Math.min(...data.map((d) => d.low ?? d.avg), 0);
+  const highestChartValue = Math.max(...data.map((d) => d.high ?? d.avg), 0);
```

Comment:
Domyślam się, że dodałeś 0 dla sytuacji gdy nie ma danych do wyświetlenia, ale teraz już nie będziemy wyświetlać pustego wykresu

```suggestion
  const lowestChartValue = Math.min(...data.map((d) => d.low ?? d.avg));
  const highestChartValue = Math.max(...data.map((d) => d.high ?? d.avg));
```

Reply:
fixed